### ISSUE TITLE
At present, the erigon node is running normally for a few minutes at a time, and this problem continues to occur, which affects normal use

### ISSUE THREAD

**Issue by songqingshududu** [node-real/bsc-erigon#240]
![~_1}Q02{O(SZTV)KJ6$YBJH](https://github.com/node-real/bsc-erigon/assets/142251921/df82d349-ba77-41dc-8e7a-5fda58f3e122)
At present, the erigon node is running normally for a few minutes at a time, and this problem continues to occur, which affects normal use

**Comment by songqingshududu** [comment]
![image](https://github.com/node-real/bsc-erigon/assets/142251921/df823d75-2a43-48f1-ab1f-7945df2bd714)

**Comment by songqingshududu** [comment]
nohup /root/jiedian/bsc-erigon/build/bin/erigon --downloader.verify --datadir=/root/jiedian/kuaizhao/erigon/chaindata  --networkid=56 --chain=bsc --db.pagesize=8k --prune=hrtc --p2p.protocol=66,68 --nat=any --sentry.drop-useless-peers=true --maxpeers=200 --port=30303 --http.addr=0.0.0.0 --http.port=1520 --authrpc.addr=0.0.0.0 --authrpc.port=1521 --torrent.port=42069 --private.api.addr=127.0.0.1:1522 --http --ws --http.api=eth,debug,net,trace,web3,erigon  --log.dir.path=/root/jiedian >> erigon.log 2>&1 &

**Comment by songqingshududu** [comment]
![image](https://github.com/node-real/bsc-erigon/assets/142251921/044ff4f7-654b-4bca-bc32-8d50d22edb86)
It works fine for a few minutes at a time, and then it's like: No block bodies to write in this log period block number=, and then I'm stuck in this place all the time, so what do I do

**Comment by songqingshududu** [comment]
目前我是按照：https://github.com/node-real/bsc-erigon/issues/234  ，来执行的

**Comment by songqingshududu** [comment]
i cry i cry i cry  T_T  T_T T_T T_T T_T T_T

**Comment by songqingshududu** [comment]
> 看来没卡住啊 在主体下载阶段结束之前只需花费几秒钟。

In fact, he would repeat the operation for an hour before it was fixed. After the repair runs normally for ten to twenty minutes, it will start to repair again, and repair for about an hour. Keep repeating these actions

**Comment by presIey** [comment]
Yes, I got the same problem,I don't know how to solve it. Maybe we need a new snapshots to download.

**Comment by fylsan3** [comment]
My node in Hong Kong has suddenly encountered this issue in the past two weeks, and I have not performed any version updates, restarts, or other operations

**Comment by fylsan3** [comment]
@songqingshududu 你们节点在哪部署着，我怀疑可能官方有什么变动导致的不稳定。我在亚马逊上长期有一个归档节点也是这种现象，我的香港机房同样如此

**Comment by songqingshududu** [comment]
> @songqingshududu你们节点在哪部署着，我怀疑可能官方有什么波动导致不稳定。我在亚马逊上长期有一个归档节点也是这种现象，我的香港机房也如此




> 重现同样的问题，将做更多测试

**Comment by danlaav1** [comment]
@blxdyx Same issue. Restarting the node solves it, but at the cost of losing synchronization. It reproduces only when the node is synced; when it's not synced, the synchronization progress goes without troubles.
